packages/website/posts/2022-01-25-what-am-buying.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'When you buy an NFT, what are you actually getting?'
+description: Behind the glamour, NFTs are more like hyperlinks than like files.
+author: J Chris Anderson
+thumbnail: https://user-images.githubusercontent.com/22168118/149502173-d50d7e98-29a9-4c53-9dbc-ed19ab0ae755.png
+date: Jan 25, 2022
+tags:
+  - culture
+---
+
```

Comment:
Might be worth sticking a paragraph about what NFTs are at their core - a unique virtual asset that links to somewhere else - might contextualize the first paragraph for readers